cloudflare/resource_cloudflare_load_balancer_monitor_test.go

```diff
@@ -115,6 +115,44 @@ func TestAccCloudflareLoadBalancerMonitor_TcpFullySpecified(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareLoadBalancerMonitor_PremiumTypes(t *testing.T) {
+	t.Parallel()
+	var loadBalancerMonitor cloudflare.LoadBalancerMonitor
+	name := "cloudflare_load_balancer_monitor.test"
```

Comment:
let's use randomly generated resource names to prevent any leaky tests or collisions in resource names (you'll need to pass this to the methods that builds the resources too).

```suggestion
	rnd := generateRandomResourceName()
	name := fmt.Sprintf("cloudflare_load_balancer_monitor.%s", rnd)
```

Reply:
sure, I'll make that alteration